configs/config.pcengines_apu2

```diff
@@ -9,5 +9,5 @@ CONFIG_BUILD_IPXE=y
 CONFIG_PXE_ROM_ID="8086,157b"
 CONFIG_MEMTEST_SECONDARY_PAYLOAD=y
 CONFIG_CPU_MICROCODE_CBFS_NONE=y
-CONFIG_PXE_CUSTOM_GENERAL_H="general.h"
```

Comment:
No more embedding scripts into iPXE?

Reply:
@miczyg1 yes, please note that I will set this as default in config options, this limit number of custom changes that mainline have to accept